examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -176,6 +205,127 @@ std::unique_ptr<GeneratedCluster> CreateCluster(chip::ClusterId id)
     return nullptr;
 }
 
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const Span<const char> & v)
+{
+    return wr.PutString(chip::TLV::Tag(), v);
+}
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const bool & v)
```

Comment:
nit: add a newline before function implementation. Seems to be more consistently spaced.

Reply:
Also, after looking at more code, why are these called "ReadValueFromBuffer" ? This function for example has absolutely no buffer in it ... it has a "writer" and a "bool". 

It also says "ReadValue" and it performs a "put" using a "writer" .... that seems also odd naming.

what is this? `WriteTLVValue`? Something else? Add a comment on one of the instances explaing the purpose and usage.

Reply:
Renamed function in [b91b1f8](https://github.com/project-chip/connectedhomeip/pull/23840/commits/b91b1f831352c37876d759ee883df7802abe9089).